treeherder/webapp/api/resultset.py

```diff
@@ -152,9 +89,6 @@ def list(self, request, project, jm):
             'results': results,
```

Comment:
If you don't have any records in a repo, then ``for rs in objs`` will never initialize ``results`` and it'll die with a 503 on this endpoint.  Should perhaps just initialize ``results = {}`` before that for-loop.